Sources/Quick/DSL/DSL.swift

```diff
@@ -65,15 +65,15 @@ public func sharedExamples(_ name: String, closure: @escaping SharedExampleClosu
     - parameter closure: A closure that can contain other examples.
     - parameter flags: A mapping of string keys to booleans that can be used to filter examples or example groups.
 */
-public func describe(_ description: String, flags: FilterFlags = [:], closure: () -> Void) {
-    World.sharedWorld.describe(description, flags: flags, closure: closure)
+public func describe(_ description: String, order: Order = Order.defined, flags: FilterFlags = [:], closure: () -> Void) {
+    World.sharedWorld.describe(description, order: order, flags: flags, closure: closure)
 }
 
 /**
     Defines an example group. Equivalent to `describe`.
 */
-public func context(_ description: String, flags: FilterFlags = [:], closure: () -> Void) {
-    World.sharedWorld.context(description, flags: flags, closure: closure)
+public func context(_ description: String, order: Order = Order.defined, flags: FilterFlags = [:], closure: () -> Void) {
```

Comment:
Nit: I would avoid duplicating the type in all these: `order: Order = .defined`.